scout/load/hgnc_gene.py

```diff
@@ -16,6 +17,22 @@
 LOG = logging.getLogger(__name__)
 
 
+def set_gene_coordinates(gene_data: dict, cytoband_coords: Dict[str, dict]):
```

Comment:
As with the comment for the call, you are afaik right about the other coord source being ensembl, but it would feel more straightforward to check for the existence of good values on chr, start stop? Regardless, perhaps change to the function name to e.g. `set_missing_gene_coordinates`, `set_empty_gene_coords_from_cytoband_location` or such.